src/index.js

```diff
@@ -94,3 +110,53 @@ function generateTripCosts(traveler) {
   let includingAgentFee = (totalSpent * 1.1);
   return includingAgentFee
 }
+
+function generateNewTrip() {
+
+
+}
+
+function validateDateEntry() {
+  let validDate;
+  let dateInput = document.querySelector('.date-input')
+  if(moment(dateInput.value)._isValid || moment(dateInput.value).isAfter(moment(Date.now()))) {
+    validDate = true
+  } else {
+    validDate = false
+  }
+  return validDate
+}
```

Comment:
I like how you've broken out these validation methods. Mine's a monster that needs refactoring.